src/menu/rendersettingsmenustate.cpp

```diff
@@ -369,54 +383,31 @@ std::unique_ptr<MenuState>
       "%1%x Anisotropic Filtering", world.getEngine().getEngineConfig()->renderSettings.anisotropyLevel));
   }
 
-  {
-    int maxW = 0;
-    int totalH = 0;
-    static constexpr int Separation = 10;
-    for(const auto& listBox : m_listBoxes)
-    {
-      listBox->fitToContent();
-      maxW = std::max(maxW, listBox->getSize().x);
-      totalH += listBox->getSize().y + Separation;
-    }
-
-    int y = ui.getSize().y - totalH - 90;
-    for(const auto& listBox : m_listBoxes)
-    {
-      const auto height = listBox->getSize().y;
-      listBox->setSize({maxW, height});
-      listBox->setPosition({(ui.getSize().x - maxW) / 2, y});
-      y += height + Separation;
-    }
-  }
+  m_tabs->fitToContent();
+  m_tabs->setPosition({(ui.getSize().x) / 2 - (m_tabs->getSize().x / 2), ui.getSize().y -m_tabs->getSize().y - 90});
 
   for(size_t i = 0; i < m_listBoxes.size(); ++i)
   {
     const auto& listBox = m_listBoxes[i];
-    listBox->update(i == m_currentListBox);
-    listBox->draw(ui, world.getPresenter());
+    listBox->update(i == m_tabs->getSelectedTab());
```

Comment:
```suggestion
    m_listBoxes[i]->update(i == m_tabs->getSelectedTab());
```
the temporary variable `const auto& listBox = m_listBoxes[i];` can then be removed